grpc-example/grpc-example-java/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```

Comment:
gRPC samples [used to](https://github.com/lagom/lagom-samples/pull/104) have some issue with sbt `1.3.x`. I hope it's fixed now, but we may need to bump `akka-grpc`/`play-grpc` too (or stay on `1.2.8`).